pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
+
+
+    def get_full_item_name(self, search_text: str, site: str) -> str:
+        if not search_text:
+            return None
+
+        try:
+            if site not in ["market", "wiki"]:
+                raise ValueError("Invalid site. Choose 'market' or 'wiki'.")
+
+            search_url = f"https://www.google.com/search?q=tarkov+{site}+" + urllib.parse.quote_plus(search_text)
+            page = requests.get(search_url)
```

Ruff (0.8.0)

Probable use of `requests` call without timeout

(S113)